CDLUC3/dmptool
# Fix for issue uploading Logos

Fixes #620.

Changes proposed in this PR:
- We installed the lighter weight GraphicsMagick on the servers
- Add binstubs for the ImageMagick `convert` and `identify` commands
- Update Dragonfly config to tell it to use the binstubs

We will need to schedule time with @ashleygould to get GraphicsMagick installed on the prod hosts

To test:
- If logging in as an organizational admin
  - Go to the Admin -> Organization details menu item
- If logging in as a super admin
  - Go to the Admin -> Organizations menu item
  - Edit an Org
  - Make sure the org has a contact email
- Replace or Add a logo
- Click Save
- The logo should appear and no error message should be present
